pkg/util/yamlutil/node_tools.go

```diff
@@ -22,6 +22,15 @@ func YamlErrorf(node *yaml.Node, format string, a ...interface{}) error {
 // Get attempts to obtain the value at an index in a sequence or in a mapping. Returns the node and true if successful;
 // nil and false if out of bounds or not found; and nil, false, and an error if any error occurs.
 func Get(l *yaml.Node, key interface{}) (*yaml.Node, bool, error) {
+	contains := func(s []string, str string) bool {
```

Comment:
I think this might actually be a bug today - nothing uses this function right now, but before this change, it fails to find anything I'd expect it to match on.